frontend/tests/unit/components/shasIntake/EnquiryIntakeForm.spec.ts

```diff
@@ -4,7 +4,7 @@ import ConfirmationService from 'primevue/confirmationservice';
 import ToastService from 'primevue/toastservice';
 import Tooltip from 'primevue/tooltip';
 import { submissionService } from '@/services';
-import { mount, RouterLinkStub } from '@vue/test-utils';
+import { flushPromises, mount, RouterLinkStub } from '@vue/test-utils';
 import { nextTick } from 'vue';
 import type { AxiosResponse } from 'axios';
 import EnquiryIntakeForm from '@/components/housing/enquiry/EnquiryIntakeForm.vue';
```

Comment:
Style: Double check the import ordering and groupings - IIRC first group is for external imports, second group is for internal code groups, and third is only for import typings.